be/src/exec/pipeline/pipeline_driver_poller.h

```diff
@@ -48,6 +48,12 @@ class PipelineDriverPoller {
     void add_blocked_driver(const DriverRawPtr driver);
     // remove blocked driver from poller
     void remove_blocked_driver(DriverList& local_blocked_drivers, DriverList::iterator& driver_it);
+
+    // add parked driver to poller
+    void add_parked_driver(const DriverRawPtr driver);
```

Comment:
how about just `park_driver`? since it's a verb

Reply:
done